password/src/Data/Password/Argon2.hs

```diff
@@ -326,6 +331,20 @@ parseAll argon2Variant argon2Version parametersT salt64 hashedKey64 = do
             ("p=", i) -> go xs (m, t, readT i)
             _ -> Nothing
 
+-- | Extract 'Argon2Params' from a 'PasswordHash' 'Argon2'.
+--
+-- Returns 'Just Argon2Params' on success.
+--
+-- >>> let pass = mkPassword "foobar"
+-- >>> passHash <- hashPassword pass
+-- >>> isJust $ extractParams passHash
```

Comment:
Best to change this to

```hs
-- >>> let pass = mkPassword "foobar"
-- >>> passHash <- hashPassword pass
-- >>> extractParams passHash == Just defaultParams
-- True
```
You don't need the `isJust` and gives a better idea that the correct parameters are extracted.